packages/cli/src/lib/project/project.spec.ts

```diff
@@ -0,0 +1,113 @@
+jest.mock('@coveord/platform-client');
+jest.mock('fs');
+jest.mock('archiver');
+
+import {mocked} from 'ts-jest/utils';
+import {existsSync, createWriteStream, WriteStream, unlinkSync} from 'fs';
+import {Project} from './project';
+import {join} from 'path';
+import * as archiver from 'archiver';
```

Comment:
Mh, I would enable `esModuleInterop` in the tsconfig and do 
```suggestion
import archiver, {Archiver} from 'archiver';
```

Reply:
I don't think it will work in this case since there are no default export in the archiver package

Reply:
![image](https://user-images.githubusercontent.com/12366410/120543515-602fa280-c3ba-11eb-9dc1-7c7462769183.png)
Tested on my machine, it works ;)
UT green too
![image](https://user-images.githubusercontent.com/12366410/120544265-375bdd00-c3bb-11eb-9de9-379b16938b29.png)

Reply:
oups, I updated the wrong tsconfig 🤦